File: backend/src/main/java/com/vencent/hotelmanagement/entity/Role.java
package com.vencent.hotelmanagement.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;
import javax.persistence.Id;
import javax.persistence.GeneratedValue;


@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name="role")
public class Role {

	@Id
	@GeneratedValue
	@Column(name="role_id")
	private int roleId;
	
	@Column(name="role_name")
	private String roleName;

	public int getRoleId() {
		return roleId;
	}

	public void setRoleId(int roleId) {
		this.roleId = roleId;
	}

	public String getRoleName() {
		return roleName;
	}

	public void setRoleName(String roleName) {
		this.roleName = roleName;
	}
	
}

File: frontend/src/app/shared/class/role.ts
export class Role {
    role_id: string;
    role_name: string;

    constructor(role_id: string, role_name: string){
        this.role_id = role_id;
        this.role_name = role_name;

    }

    set name(Name: string){
        this.role_name = Name;
    }

    get name(): string{
        return this.name;
    }

}
